internal/service/autoscaling/policy.go

```diff
@@ -681,6 +955,94 @@ func expandPredefinedScalingMetricSpecification(predefinedScalingMetricSpecifica
 	return predefinedScalingMetricSpecification
 }
 
+func expandCustomizedScalingMetricSpecification(customizedScalingMetricSpecificationSlice []interface{}) *autoscaling.PredictiveScalingCustomizedScalingMetric {
+	if customizedScalingMetricSpecificationSlice == nil || len(customizedScalingMetricSpecificationSlice) < 1 {
+		return nil
+	}
+	customizedScalingMetricSpecificationFlat := customizedScalingMetricSpecificationSlice[0].(map[string]interface{})
+	customizedScalingMetricSpecification := &autoscaling.PredictiveScalingCustomizedScalingMetric{
+		MetricDataQueries: expandMetricDataQueries(customizedScalingMetricSpecificationFlat["metric_data_queries"].([]interface{})),
+	}
+	return customizedScalingMetricSpecification
+}
+
+func expandCustomizedLoadMetricSpecification(CustomizedLoadMetricSpecificationSlice []interface{}) *autoscaling.PredictiveScalingCustomizedLoadMetric {
+	if CustomizedLoadMetricSpecificationSlice == nil || len(CustomizedLoadMetricSpecificationSlice) < 1 {
+		return nil
+	}
+	CustomizedLoadMetricSpecificationSliceFlat := CustomizedLoadMetricSpecificationSlice[0].(map[string]interface{})
```

Comment:
Capitalization should be consistent for variable names.

`customizedLoadMetricSpecificationSliceFlat`